src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -168,15 +176,36 @@ internal void BeginConnection(Guid nodeId, int portIndex, PortType portType)
             }
         }
 
+        internal void BeginShiftReconnections(Guid nodeId, int portIndex, PortType portType)
+        {
+            if (portType == PortType.Input) return;
+
+            NodeModel node = Model.GetModelInternal(nodeId) as NodeModel;
+            if (node == null) return;
+
+            PortModel portModel = node.OutPorts[portIndex];
+            if (portModel.Connectors.Count <= 0) return;
+            
+            var connectorsAr = new ConnectorViewModel[portModel.Connectors.Count];
+            var c = new ConnectorViewModel(this, portModel.Connectors[0].End);
```

Comment:
This initialization is not needed as it's been overwritten at i = 0 inside for loop.